packages/vaex-ml/vaex/ml/transformations.py

```diff
@@ -8,6 +8,9 @@
 import numpy as np
 import warnings
 
+sklearn = vaex.utils.optional_import("sklearn.decomposition")
+sklearn = vaex.utils.optional_import("sklearn.random_projection")
```

Comment:
The 1st statement doesn't do anything, since the 2nd overwrite it, but I understand what you mean. But you do not refer to decomposition anyway, so we can just remove it right?
```suggestion
sklearn = vaex.utils.optional_import("sklearn.random_projection")
```